index.js

```diff
@@ -47,6 +47,8 @@ module.exports = {
     "new-cap": 0,
     "comma-spacing": [2, {"before": false, "after": true}],
     "comma-dangle": [2, "always-multiline"],
+    "eqeqeq": [2, "always"],
```

Comment:
So I did some [research](http://adripofjavascript.com/blog/drips/equals-equals-null-in-javascript.html) into this rule and actually learned something useful: `x == null` tests true for x values of `null` and `undefined`, but only those two, which seems useful.  Maybe we can also pass in a `"null"` value of the rule with `"ignore"`